src/Symfony/Component/Mailer/Bridge/MailerSend/Tests/Transport/MailerSendTransportFactoryTest.php

```diff
@@ -0,0 +1,90 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Mailer\Bridge\MailerSend\Tests\Transport;
+
+use Symfony\Component\Mailer\Bridge\MailerSend\Transport\MailerSendApiTransport;
+use Symfony\Component\Mailer\Bridge\MailerSend\Transport\MailerSendSmtpTransport;
+use Symfony\Component\Mailer\Bridge\MailerSend\Transport\MailerSendTransportFactory;
+use Symfony\Component\Mailer\Test\TransportFactoryTestCase;
+use Symfony\Component\Mailer\Transport\Dsn;
+use Symfony\Component\Mailer\Transport\TransportFactoryInterface;
+
+class MailerSendTransportFactoryTest extends TransportFactoryTestCase
+{
+    public static function getFactory(): TransportFactoryInterface
+    {
+        return new MailerSendTransportFactory(self::getDispatcher(), self::getClient(), self::getLogger());
```

Comment:
You may change this line, like this one https://github.com/symfony/symfony/pull/49431/files#diff-fa09ff906f695689d8b225fd97ac12dc65bc9f205e0600380ec96e67f204ae3aR30 :slightly_smiling_face:

Reply:
Thanks! Updated this line and more on `createProvider` method.

Reply:
Great!